src/patternfly/components/Button/styles.scss

```diff
@@ -184,6 +191,7 @@
 
   // Primary Buttons
   &.pf-m-primary {
+    font-weight: var(--pf-c-button--m-primary--FontWeight);
```

Comment:
Can't we define this on `.btn` as `--pf-c-button--FontWeight` so all btns are semibold instead of defining them on each modifier?